textdb/textdb-dataflow/src/main/java/edu/uci/ics/textdb/dataflow/common/JoinDistancePredicate.java

```diff
@@ -107,6 +109,53 @@ public String getJoinAttributeName() {
     public Integer getThreshold() {
         return this.threshold;
     }
+    
+    public Schema generateOutputSchema(Schema innerOperatorSchema, Schema outerOperatorSchema) throws DataFlowException {
+        return generateIntersectionSchema(innerOperatorSchema, outerOperatorSchema);
```

Comment:
the `generateIntersectionSchema` function is moved from `Join` to `JoinDistancePredicate`